src/main.rs

```diff
@@ -1,249 +1,26 @@
-use ansi_term::Color::{Black, Red};
+use ansi_term::Colour::{Black, Red};
```

Comment:
We have switched to `yansi` recently.

Reply:
My bad i did see the pr for yansi but forgot about it. Removed all ansi_term and used yansi instances